templates/apiModels.mustache

```diff
@@ -59,10 +59,10 @@ export interface FieldConfig {
      */
     maximum: number | undefined;
 	/**
-	 * Options available if the fieldType is of type {@link FieldType.ENUM}.
-     * Undefined if the fieldType is not of type {@link FieldType.ENUM}.
+	 * Options available if the fieldType is of type {@link FieldType.ENUM} or {@link FieldType.ARRAY}.
+     * Undefined if the fieldType is not of type {@link FieldType.ENUM} or type {@link FieldType.ARRAY}.
 	 */
-	options: Array<string> | undefined;
+	options: Array<any> | undefined;
```

Comment:
- [ ] Is `any` correct? I think we can get away with `string | number`?

Reply:
Updated.